### PR TITLE
Feat: enabling email notifications for failed dags

### PR DESCRIPTION
**Pull request by pbastia**
This PR is to enable email notifications for failed DAGs

### CODE REVIEW

File: dags/automated_email_test_dag.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+
+# -*- coding: utf-8 -*-
+"""
+# DAG triggering the cas-ggircs-acme-renewal cron job
+"""
```
Comment (matthieu-foucault) [inline dags/automated_email_test_dag.py:5]: The DAG description needs an update

File: helm/cas-airflow/templates/cronJobs/test-email-on-failure-cronjob.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+    "helm.sh/hook-weight": "-5"
+spec:
+  suspend: true # This cron job is intended to be triggered manually
+  schedule: "* * * * *"
+  jobTemplate:
+    spec:
+      backoffLimit: 0
+      activeDeadlineSeconds: 600
+      ttlSecondsAfterFinished: 10
```
Comment (matthieu-foucault) [inline helm/cas-airflow/templates/cronJobs/test-email-on-failure-cronjob.yaml:18]: Did we establish if this is supported by OCP4?

File: helm/cas-airflow/templates/cronJobs/test-email-on-failure-cronjob.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+              image: gcr.io/ggl-cas-storage/alpine:3.13.5
+              resources:
+                requests:
+                  cpu: 100m
+                  memory: 32Mi
+              command: 
+                - /usr/bin/env
+                - sh
+                - -c
+                - echo "This CronJob purposely fails, airflow should report an error to ggircs@gov.bc.ca" && exit 1
```
Comment (matthieu-foucault) [inline helm/cas-airflow/templates/cronJobs/test-email-on-failure-cronjob.yaml:37]: add newline at end of file

**Review by matthieu-foucault — CHANGES_REQUESTED** [review]
Looks good, just some minor changes needed

**Review by matthieu-foucault — APPROVED** [review]

File: dags/automated_email_test_dag.py (hunk 1 above)
Comment (pbastia) [reply dags/automated_email_test_dag.py]: oops

**Pull request merged**